api/resources_portal/test/test_data_readme.txt

```diff
@@ -0,0 +1,45 @@
+The test data describes the following situation:
+
+There are three users:
+- PrimaryProf
+- PostDoc
+- SecondaryProf
+
+There are fours organizations which have these users as members:
+- PrimaryLab, owned by PrimaryProf
+- PrimaryProfPersonalOrg, owned by PrimaryProf with PostDoc as a member
+- PostDocOrg, owned by PostDoc
+- SecondaryProfOrg, owned by SecondaryProf
+
+There are two grants:
+- A grant for melanoma research, given to PrimaryProf
+- A grant for tumor growth research, given to SecondaryProf
+
+There are eight materials:
+	Four contributed by PrimaryProf:
+	- A Plasmid
+	- A Protocol
+	- A Model Organism (zebrafish)
+	- A Model Organism (mouse)
+
+	Four contributed by SecondaryProf:
+	- A Stone Tablet (using the "other" template)
+	- A Cell Line
+	- A Dataset
+	- A PDX
+
+There are the following permission groups:
+- Admin
+- User
+
+The users are in the following groups:
+- PrimaryProf: Admin
+- SecondaryProf: Admin
+- PostDoc: User
```

Comment:
I think all of these users should be `User` not `Admin`. If I'm not mistaken `Admin` is for superusers, like us if we had an account and wanted to modify data via the admin panel.

The permission difference between the Postdoc and the PrimaryProf comes from the fact that the PrimaryProf owns his organization so he'll have all the permissions for it including changing permissions and such. I think the permission system and authentication system are different.